chapters/appendix-IX-file-tags.md

```diff
@@ -0,0 +1,51 @@
+# Appendix IX: SPDX File Tags
+
+## Rationale
+
+SPDX short-form license identifiers using the [`SPDX-License-Identifier:`](https://spdx.org/ids) tag, as described in [Appendix V](./appendix-V-using-SPDX-short-identifiers-in-source-files.md), provide a mechanism for developers to easily convey information about the licenses they declare on a file-by-file basis. That mechanism is intentionally very easy for software tools to identify and detect, since it includes a standard text string that is unlikely to occur in other contexts, and since it uses license identifiers from the [SPDX License List](https://spdx.org/licenses) or from user-defined `LicenseRef-` statements.
+
```

Comment:
are we specifically/explicitly recommending user-defined LicenseRef- identifiers be used with SPDX-License-Identifier: ? that seems new - not that I'm opposed, but it could confuse things to state it here?

Reply:
This was recently explicitly added to Appendix V in #207 -- to reflect officially that this is considered valid.

The addition there does explicitly state that if you're going to do it, you also have to make sure someone has a way to determine what the LicenseRef- corresponds to, and it notes that REUSE provides a way to do this (see https://reuse.software/spec/)